docker.nix

```diff
@@ -0,0 +1,99 @@
+{ dockerTools, backend, frontend, runCommand, writeText, writeScriptBin, bash, cacert, gettext, iana_etc, caddy }:
+let
+  inherit (dockerTools)
+    buildImage
+    buildLayeredImage
+    pullImage
+    shadowSetup;
+
+  inherit (builtins) toJSON;
+
+  httpPort = 80;
+  httpsPort = 443;
+
+  # Some system files necessary for nginx to function normally
+  shadow-files = runCommand "shadow-files" {} ''
+    mkdir -p $out/etc
+    echo "nobody:x:65534:65534:nobody:/:/sbin/nologin" > $out/etc/passwd
+    echo "nobody:!x:::::::" > $out/etc/shadow
+    echo "nobody:x:65534:" > $out/etc/group
+    echo "nobody:x::" > $out/etc/gshadow
+  '';
+
+  backend-config-template = writeText "backend-config-env.yaml.template" (toJSON {
+    api.listen-addr = "*:$API_PORT";
+    api.serve-docs = "$SERVE_DOCS";
+  });
+
+  backend-entry-point = writeScriptBin "entrypoint.sh" ''
+    #!/bin/bash
+    set -euxo pipefail
+
+    /bin/envsubst '$API_PORT $SERVE_DOCS' \
+      < ${backend-config-template} >| /config.yaml
+
+    exec /bin/edna-server -c -c /config.yaml "$@"
```

Comment:
Looks like redundant duplicated `-c`.

Reply:
I have no idea how it is run or configured yet :stuck_out_tongue: I need to have a look at that.